### PR TITLE
Enhance docs and add autobuild

### PR DESCRIPTION
**Pull request by kirkbrauer**

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -13,87 +13,87 @@ files = [
 
 [[package]]
 name = "aiohttp"
-version = "3.10.1"
+version = "3.10.3"
```
Comment (mangelajo) [inline poetry.lock:16]: did this one sneak in?, anyway not a big deal :)

Comment (kirkbrauer) [reply poetry.lock:16]: Humm something might have forced the lock file to update when I ran `poetry lock` after rebasing.

**Comment by mangelajo** [comment]
@kirkbrauer I see it's marked as draft, but does it make to merge and iterate later? :)

**Review by mangelajo — APPROVED** [review]

**Comment by mangelajo** [comment]
🚀 🙇

**Pull request merged**